include/iota/api/responses/base.hpp

```diff
@@ -71,8 +71,19 @@ class Base {
    */
   void setDuration(const int64_t& duration);
 
+  /**
+   * @return status code of operation
+   */
+  const int16_t& getStatusCode() const;
+
+  /**
+   * @param statusCode set statusCode of operation
+   */
+  void setStatusCode(const int16_t& statusCode);
+
 private:
   int64_t duration_;
+  int16_t statusCode_;
```

Comment:
can you explicitly initialize this one to 0 in the ctor init list, otherwise we will have some potential uninitialized scalar value.